crates/miden-tx/src/prover/mod.rs

```diff
@@ -118,35 +117,29 @@ impl LocalTransactionProver {
 
     pub fn prove(
         &self,
-        tx_witness: TransactionWitness,
+        executed_transaction: ExecutedTransaction,
     ) -> Result<ProvenTransaction, TransactionProverError> {
```

Comment:
Do we actually need to pass the full `ExecutedTransaction` here? Seems like just `TransactionInputs` could be sufficient.